src/modules/nmx/NMXBase.cpp

```diff
@@ -209,6 +209,14 @@ void NmxBase::processing_thread() {
 
       for (auto &builder : NMX.builders) {
         NMX.generateEvents(builder.Events);
+        if (NMX.Conf.NMXFileParameters.SplitMultiEvents) {
+          Counters.EventsSpanTooLarge += builder.matcher.Stats.SpanTooLarge;
```

Comment:
This can be done more elegantly with 

Counters.Matcher = builder.matcher.Stats if we keep the stats in a struct